src/Core/Managed/PCL/Extensibility/Implementation/Platform/PlatformImplementation.cs

```diff
@@ -35,5 +35,10 @@ public IDebugOutput GetDebugOutput()
             
             return this.debugOutput;
         }
+
+        public string GetEnvironmentVariable(string name)
+        {
+            return null;
```

Comment:
Not in PCL. Those are .Net Core specific namespaces. If ever change target framework we can start supporting it.